2select-html/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="styles.css" rel="stylesheet" />
+    <title>Selecting HTML Elements to Style with CSS</title>
+  </head>
+  <body>
+    <!-- the body element is the parent, the header element is the child -->
+    <header>
+      <!-- header and article elements are siblings since they next in the body -->
+      <h1>About Coral Reefs</h1>
+    </header>
+    <article>
+      <header>
+        <h2>Coral Reef <strong>Biodiversity</strong></h2>
+      </header>
+      <section>
+        <p>
+          Coral reefs are teeming with life. They are known as the
+          <em>rainforests of the sea</em> with how many various species live
+          within their waters. The defining feature of these ecosystems are the
+          plant-like coral, which are really colonies of tiny invertebrates
+          called <strong>polyps</strong>.
+        </p>
+        <p>
+          Sadly, many reefs around the world are in danger due to rising ocean
+          temperatures, pollution, and overfishing.
+        </p>
+      </section>
+      <section>
+        <h3><strong>Animal Life</strong> in a Coral Reef</h3>
+        <ul>
+          <li>Angelfish</li>
+          <li>Clownfish</li>
+          <li>Octopus</li>
+          <li><strong>Sharks</strong></li>
+          <li>Barracuda</li>
+        </ul>
+      </section>
+      <section><h3>Sammy's <strong>Favorite</strong> Reef Food</h3>
+    <ol>
+      <li>Sea Grass</li>
+      <li><strong>Kelp</strong></li>
+      <li>Sea Grapes</li>
+      <li>Sea Lettuce</li>
+    </ol></section>
```

Comment:
You likely want this `</section>` closing tag on it's own line, to properly format your html "nesting".